ansible/roles/pkg-jdk8/README.md

```diff
@@ -0,0 +1,16 @@
+Package: jdk8
+=============
+
+Install jdk8
+
+- FreeBSD (install openjdk8)
+
+Requirements
+------------
+
+None.
+
+Author Information
+------------------
+
+ome-devel@lists.openmicroscopy.org.uk
```

Comment:
I'd vote for a longer name, e.g. `java-jdk8` but not a blocker for an internal role, so I'll let others decide.